packages/nx/src/tasks-runner/run-command.ts

```diff
@@ -374,6 +484,41 @@ async function promptForApplyingSyncGeneratorChanges(): Promise<boolean> {
   }
 }
 
+async function confirmRunningTasksWithSyncFailures(): Promise<void> {
+  try {
+    const promptConfig = {
+      name: 'runTasks',
+      type: 'select',
+      message:
+        'Would you like to ignore the sync failures and continue running the tasks?',
+      choices: [
+        {
+          name: 'yes',
+          message: 'Yes, ignore the failures and run the tasks',
+        },
+        {
+          name: 'no',
+          message: `No, don't run the tasks`,
+        },
+      ],
+      footer: () =>
+        chalk.dim(
+          `\nWhen running on CI and there are sync failures, the tasks won't run. Addressing the errors above is highly recommended to prevent failures in CI.`
```

Comment:
```suggestion
          `\nWhen running in CI and there are sync failures, the tasks won't run. Addressing the errors above is highly recommended to prevent failures in CI.`
```

Reply:
You actually had both "on" and "in" in the same message here :D

Reply:
You don't know the challenge it poses "in" vs. "on" to non-native English speakers 😅